test.js

```diff
@@ -0,0 +1,7 @@
+function hello(name) {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'name' is defined but never used  no-unused-vars